examples/node/vector_search_example.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+import { Decoder, GlideClient, GlideClusterClient, GlideFt, FtSearchOptions,
+    FtSearchReturnType, Logger, VectorField} from "@valkey/valkey-glide";
+
+import { v4 as uuidv4 } from "uuid";
+
+const DATA_PROCESSING_TIMEOUT = 1000;
+
+async function executeVssCommands() {
+    // When Valkey is in standalone mode, add address of the primary node, and any replicas you'd like to be able to read from.
+    // JSON modules and VSS modules must be enabled for this to successfully execute, otherwise there will be request errors. 
+    const addresses = [
+        {
+            host: "localhost",
+            port: 6380,
+        },
+    ];
+    // Check `GlideClientConfiguration/GlideClusterClientConfiguration` for additional options.
+    const client = await GlideClient.createClient({
+        addresses: addresses,
+        // if the server uses TLS, you'll need to enable it. Otherwise the connection attempt will time out silently.
+        // useTLS: true,
+        clientName: "test_standalone_client",
+    });
+    await vssCreateAndSearch(client);
+    client.close();
+}
+
+async function vssCreateAndSearch(client: GlideClient | GlideClusterClient) {
+    const prefix = "{" + uuidv4() + "}:";
+    const index = prefix + "index";
+    const query = "*=>[KNN 2 @VEC $query_vec]";
+    
+    const vectorField_1: VectorField =  {
+        type: "VECTOR",
```

Comment:
This is a default field, no need to pass this.

Reply:
it doesn't work without passing in `type: "VECTOR",`

Reply:
The `"VECTOR"` value in type declaration of `VectorField` is not a "default value", but a "literal type".